config/log4j2.properties

```diff
@@ -0,0 +1,163 @@
+appender.stateChangeAppender.layout.type=PatternLayout
+appender.stateChangeAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.stateChangeAppender.policies.type=TimeBasedTriggeringPolicy
+appender.stateChangeAppender.policies.interval=1
+appender.stateChangeAppender.policies.modulate=true
+
+# Request appender
+appender.requestAppender.type=RollingFile
+appender.requestAppender.name=RequestAppender
+appender.requestAppender.fileName=${kafka.logs.dir}/kafka-request.log
+appender.requestAppender.filePattern=${kafka.logs.dir}/kafka-request.log.%d{yyyy-MM-dd-HH}
+appender.requestAppender.layout.type=PatternLayout
+appender.requestAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.requestAppender.policies.type=TimeBasedTriggeringPolicy
+appender.requestAppender.policies.interval=1
+appender.requestAppender.policies.modulate=true
+
+# Cleaner appender
+appender.cleanerAppender.type=RollingFile
+appender.cleanerAppender.name=CleanerAppender
+appender.cleanerAppender.fileName=${kafka.logs.dir}/log-cleaner.log
+appender.cleanerAppender.filePattern=${kafka.logs.dir}/log-cleaner.log.%d{yyyy-MM-dd-HH}
+appender.cleanerAppender.layout.type=PatternLayout
+appender.cleanerAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.cleanerAppender.policies.type=TimeBasedTriggeringPolicy
+appender.cleanerAppender.policies.interval=1
+appender.cleanerAppender.policies.modulate=true
+
+# Controller appender
+appender.controllerAppender.type=RollingFile
+appender.controllerAppender.name=ControllerAppender
+appender.controllerAppender.fileName=${kafka.logs.dir}/controller.log
+appender.controllerAppender.filePattern=${kafka.logs.dir}/controller.log.%d{yyyy-MM-dd-HH}
+appender.controllerAppender.layout.type=PatternLayout
+appender.controllerAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.controllerAppender.policies.type=TimeBasedTriggeringPolicy
+appender.controllerAppender.policies.interval=1
+appender.controllerAppender.policies.modulate=true
+
+# Authorizer appender
+appender.authorizerAppender.type=RollingFile
+appender.authorizerAppender.name=AuthorizerAppender
+appender.authorizerAppender.fileName=${kafka.logs.dir}/kafka-authorizer.log
+appender.authorizerAppender.filePattern=${kafka.logs.dir}/kafka-authorizer.log.%d{yyyy-MM-dd-HH}
+appender.authorizerAppender.layout.type=PatternLayout
+appender.authorizerAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.authorizerAppender.policies.type=TimeBasedTriggeringPolicy
+appender.authorizerAppender.policies.interval=1
+appender.authorizerAppender.policies.modulate=true
+
+rootLogger.level=INFO
+rootLogger.appenderRefs=stdout,kafkaAppender
+rootLogger.appenderRef.stdout.ref=STDOUT
+rootLogger.appenderRef.kafkaAppender.ref=KafkaAppender
+
+loggers=zookeeper,kafka,apacheKafka,requestLogger,networkRequestChannel,apacheKafkaController,kafkaController,logCleaner,stateChangeLogger,authorizerLogger
+
+# Zookeeper logger
+logger.zookeeper.name=org.apache.zookeeper
+logger.zookeeper.level=INFO
```

Comment:
Should we remove them?

Reply:
Yes, we should. However, I'm thinking that removing the zk configurations might be better handled as a follow-up PR, since this one is already quite large. 

WDYT ?